Formula/zig.rb

```diff
@@ -0,0 +1,34 @@
+class Zig < Formula
+  desc "Programming language designed for robustness, optimality, and clarity"
+  homepage "http://ziglang.org/"
+  url "https://s3.amazonaws.com/ziglang.org/download/0.1.1/zig-0.1.1.tar.xz"
+  sha256 "a160d14aecebead2f8c574b0dd687cdfecc53d149d521a17b89b629d8b558c94"
+
+  depends_on "cmake" => :build
+  depends_on "llvm@5" => :build
+
+  def install
+    Dir.mkdir "build"
+    Dir.chdir "build" do
+      system "cmake", "..",
```

Comment:
Probably you can simply say: `system "cmake", "..", *std_cmake_args` which should take care of the paths for you

Reply:
I get `Error: Empty installation` when I use that.

Reply:
`std_cmake_args` contains:

```
      -DCMAKE_C_FLAGS_RELEASE=-DNDEBUG
      -DCMAKE_CXX_FLAGS_RELEASE=-DNDEBUG
      -DCMAKE_INSTALL_PREFIX=#{prefix}
      -DCMAKE_BUILD_TYPE=Release
      -DCMAKE_FIND_FRAMEWORK=LAST
      -DCMAKE_VERBOSE_MAKEFILE=ON
      -Wno-dev
```

But I do not understand what your `CMAKE_PREFIX_PATH` is pointing to. `#{prefix}` will expand to `zig`'s own prefix (e.g. `/usr/local/Cellar/zig/0.1.1`), so why are you appending `llvm@5` to that?

Reply:
Actually maybe I'm calling `mkdir` strangely, let me test a bit.

Reply:
I was naively following [zig's build instructions](https://github.com/zig-lang/zig) to see how everything worked. `std_cmake_args` works great though!